UI/properties-view.cpp

```diff
@@ -1868,6 +1870,40 @@ void WidgetInfo::EditableListChanged()
 
 void WidgetInfo::ButtonClicked()
 {
+	obs_button_type type = obs_property_button_type(property);
+	const char *savedUrl = obs_property_button_url(property);
+
+	if (type == OBS_BUTTON_URL && strcmp(savedUrl, "") != 0) {
+		QUrl url(savedUrl, QUrl::StrictMode);
+		if (url.isValid() && (url.scheme().compare("http") == 0 ||
+				      url.scheme().compare("https") == 0)) {
+			QString msg(
+				QTStr("Basic.PropertiesView.UrlButton.Text"));
+			msg += "\n\n";
+			msg += QString(QTStr("Basic.PropertiesView.UrlButton.Text.Url"))
+				       .arg(savedUrl);
+
+			QString open =
+				QTStr("Basic.PropertiesView.UrlButton.OpenUrl");
+
+			QMessageBox messageBox(view->window());
```

Comment:
Should this be an `OBSMessageBox::question` instead?

Reply:
Ah good point, I keep forgetting that exists. I'll update this today.

Reply:
After further digging, `OBSMessageBox::question` doesn't support setting the label on the Yes button, so I've just set it to a standard Yes/No dialog.